qiime/plugin/template.py

```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from cookiecutter.main import cookiecutter
+
+
+def plugin_init(output_dir='.'):
```

Comment:
This will result in a subdirectory right? It doesn't template *inside* the current directory?